inputs/worker_inner:3.json

```diff
@@ -0,0 +1,153 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "repos": [],
+        "from_task_id": "{{FROM_TASK_ID}}"
+      }
+    },
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "name": "add_labels_in_dockerfile",
+      "args": {
+        "labels": "{{IMPLICIT_LABELS}}"
+      }
+    },
+    {
+      "name": "change_from_in_dockerfile"
+    },
+    {
+      "name": "add_help",
+      "required": false
+    },
+    {
+      "name": "add_dockerfile"
+    },
+    {
+      "name": "distgit_fetch_artefacts",
+      "args": {
+        "command": "{{SOURCES_COMMAND}}"
+      }
+    },
+    {
+      "args": {
+        "koji_root": "{{KOJI_ROOT}}",
+        "koji_hub": "{{KOJI_HUB}}"
+      },
+      "required": false,
+      "name": "fetch_maven_artifacts"
+    },
+    {
+      "name": "koji",
+      "args": {
+        "root": "{{KOJI_ROOT}}",
+        "target": "{{KOJI_TARGET}}",
+        "hub": "{{KOJI_HUB}}"
+      }
+    },
+    {
+      "name": "add_yum_repo_by_url",
+      "args": {
+        "repourls": []
+      }
+    },
+    {
+      "name": "inject_yum_repo"
+    },
+    {
+      "name": "distribution_scope"
+    }
+  ],
+  "prepublish_plugins": [
+    {
+      "name": "squash"
+    }
+  ],
+  "postbuild_plugins": [
+    {
+      "name": "all_rpm_packages",
+      "args": {
+        "image_id": "BUILT_IMAGE_ID"
+      }
+    },
+    {
+      "name": "tag_by_labels"
+    },
+    {
+      "name": "tag_from_config"
+    },
+    {
+      "name": "tag_and_push",
+      "args": {
+        "registries": {
+          "{{REGISTRY_URI}}": { "insecure": true }
+        }
+      }
+    },
+    {
+      "name": "pulp_push",
+      "args": {
+        "pulp_registry_name": "{{PULP_REGISTRY_NAME}}",
+        "load_exported_image": true,
+        "dockpulp_loglevel": "INFO"
+      }
+    },
+    {
+      "name": "pulp_sync",
+      "args": {
+        "pulp_registry_name": "{{PULP_REGISTRY_NAME}}",
+        "docker_registry": "{{DOCKER_REGISTRY}}",
+        "dockpulp_loglevel": "INFO"
+      }
+    },
+    {
+      "name": "compress",
+      "args": {
+        "load_exported_image": true,
+        "method": "gzip"
+      }
+    },
+    {
+      "name": "koji_upload",
+      "args": {
+        "kojihub": "{{KOJI_HUB}}",
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false,
+        "blocksize": 10485760,
+        "koji_keytab": false,
+        "koji_principal": false
+      }
+    },
+    {
+      "name": "pulp_pull",
+      "args": {}
+    }
+  ],
+  "exit_plugins": [
+    {
+      "name": "delete_from_registry",
```

Comment:
Nevermind. This should stay here because when it runs in orchestrator it'll be a noop until https://github.com/projectatomic/atomic-reactor/pull/741 is done.